### PR TITLE
Update Strapi to latest upstream

### PR DESCRIPTION
**Pull request by hacktivist123**

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -11,51 +11,21 @@ type: nodejs:12
...
-        fi
-        if [ -n "$(ls -A exports-tmp)" ]; then
-            cp -r exports-tmp/* exports
-        fi
-        if [ -n "$(ls -A components-tmp)" ]; then
-            cp -r components-tmp/* components
-        fi
+  build: |
+    # Download dependencies and build Strapi.
+    yarn
```
Comment (chadwcarlson) [inline .platform.app.yaml]:
```suggestion
    yarn --ignore-optional --frozen-lockfile 
```

Comment (chadwcarlson) [reply .platform.app.yaml]: You may no better than me, but it seemed like Yarn only with these flags respected the state of `yarn.lock` during a build.

Comment (hacktivist123) [reply .platform.app.yaml]: I agree with you, at the end of the day it's really about tradeoffs. I'll make the edit

Comment (hacktivist123) [reply .platform.app.yaml]: Yarn  --ignore-optional --frozen-lockfile  is actually supposed to be the default yarn behavior, I have no idea why it hasn't been considered, see this: https://github.com/yarnpkg/yarn/issues/4147

**Pull request merged**